packages/rest/src/rest.server.ts

```diff
@@ -336,11 +334,19 @@ export class RestServer extends Context implements Server, HttpServerLike {
   private async _serveOpenApiSpec(
     request: Request,
     response: Response,
-    options?: OpenApiSpecOptions,
+    settings?: OpenApiSpecOptions,
   ) {
-    options = options || {version: '3.0.0', format: 'json'};
+    settings = settings || {version: '3.0.0', format: 'json'};
     let specObj = this.getApiSpec();
-    if (options.format === 'json') {
+    if (
+      this._config.apiExplorer &&
+      this._config.apiExplorer.setServersFromRequest
```

Comment:
To avoid repetitive check for `this._config.apiExplorer`, I am proposing to always initialize this property in the constructor, so that it's either an empty object or the user-provider object. Then we can simplify this part as follows:

```ts
if (this._config.apiExplorer.setServersFromRequest)
```

See also https://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if

Reply:
The style/formatting is enforced by `prettier`.